app/views/admin/performance/support_queries/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render partial: "admin/performance/nav" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">Support form stats</h1>
+
+    <%= govuk_table do |table| %>
+      <% table.with_head do |head| %>
+        <% head.with_row do |row| %>
+          <% row.with_cell(header: true, text: "What were they trying to do?") %>
+          <% row.with_cell(header: true, text: "Support forms sent", classes: "govuk-!-text-align-right") %>
+          <% row.with_cell(header: true, text: "Number of users", classes: "govuk-!-text-align-right") %>
+        <% end %>
+      <% end %>
+
+      <% table.with_body do |body| %>
+        <% @subjects.each do |support_query_subject| %>
+          <% body.with_row(html_attributes: { data: { subject: support_query_subject }}) do |row| %>
+            <% row.with_cell(text: t("support_query.stats.#{support_query_subject}")) %>
+            <% row.with_cell(html_attributes: { data: { stat: :total } }, text: @support_query_counts[support_query_subject] || 0, numeric: true) %>
+            <% row.with_cell(html_attributes: { data: { stat: :users } }, text: @support_query_user_counts[support_query_subject] || 0, numeric: true) %>
```

Comment:
Not critical given our current low numbers, but usually worth using `number_with_delimiter` or similar to add comma separators for the 1,000s - just to make the numbers more readable?

```suggestion
            <% row.with_cell(html_attributes: { data: { stat: :total } }, text: number_with_delimiter(@support_query_counts[support_query_subject] || 0), numeric: true) %>
            <% row.with_cell(html_attributes: { data: { stat: :users } }, text: number_with_delimiter(@support_query_user_counts[support_query_subject] || 0), numeric: true) %>
```

Reply:
Absolutely, good catch